packages/component/src/Middleware/AttachmentForScreenReader/createCoreMiddleware.js

```diff
@@ -17,7 +17,7 @@ export default function createCoreMiddleware() {
         }
       ] = args;
 
-      return role === 'user' && !/^text\//u.test(contentType) && !thumbnailUrl
+      return role === 'user' && !/^text\//u.test(contentType) && !attachment.content && !thumbnailUrl
```

Comment:
I think this one is, "show as file if":

- If this is from user, or
- If this is a `text/*`
  - Show as file if it does not have `attachment.content`
- Otherwise, if this is not a `text/*`
  - Show as file if it does not have `attachment.thumbnailUrl`


```suggestion
      return role === 'user' && !/^text\//u.test(contentType) && !attachment.content && !thumbnailUrl
```

Reply:
I'm not seeing a diff.